autoload/vista/finder/fzf.vim

```diff
@@ -56,9 +56,15 @@ function! s:Run(...) abort
   let opts = {
           \ 'source': source,
           \ 'sink': function('s:sink'),
-          \ 'options': '--prompt "'.s:cur_executive.'> "',
+          \ 'options': ['--prompt', s:cur_executive.'> '] + get(g:, 'vista_fzf_opt', []),
           \ }
 
+  if exists('g:vista_fzf_preview')
+    let preview_opts = call('fzf#vim#with_preview', g:vista_fzf_preview).options
```

Comment:
I think it's more intuitive to use the option `g:vista_fzf_preview` as a String and wrap it as a List internally when calling `fzf#vim#with_preview`.

```vim
let g:vista_fzf_preview = 'right:50%'

let preview_opts = call('fzf#vim#with_preview', [g:vista_fzf_preview]).options
```

Reply:
I made `g:vista_fzf_preview` a list to support potential use cases that need multiple arguments be passed to `fzf#vim#with_preview`. I am not sure there are use cases at this time, but applying the list as arguments to `fzf#vim#with_preview` is consistent with fzf api `function! fzf#vim#with_preview(...)`. `with_preview(...)` can gain functionality without breaking it's api so we should uphold it's api.
```
command! -bang -nargs=* Rg
  \ call fzf#vim#grep(
  \   'rg --column --line-number --no-heading --color=always --smart-case '.shellescape(<q-args>), 1,
  \   <bang>0 ? fzf#vim#with_preview('up:60%')
  \           : fzf#vim#with_preview('right:50%:hidden', '?'),
  \   <bang>0)
```

Reply:
Fair enough.